src/select-list-view.ts

```diff
@@ -414,7 +441,12 @@ module.exports = class SelectListView {
 }
 
 class ListItemView {
-  constructor (props) {
+  public element: EtchElement
+  public selected: boolean
+  public onclick: () => void
+  public domEventsDisposable: Disposable
+
+  constructor (props: { element: EtchElement ; selected: boolean; onclick: () => void } ) {
```

Comment:
```suggestion
  constructor (props: { element: EtchElement; selected: boolean; onclick: () => void }) {
```

Might be worth pulling this out into its own type for clarity.

Reply:
Fixed in https://github.com/atom/atom-select-list/pull/28/commits/5abb4f0ef1e1a20ea56a4fa531da48de137718aa